README.md

```diff
@@ -39,6 +39,7 @@ Also, you can get them directly from the TypeFox Jenkins server:
 
 ### Supported DAP Versions
 
+ * LSP4J 0.7.3 &rarr; DAP 1.32.0
```

Comment:
I am not sure which version of LSP4J I should put here. @jonahgraham could you help me?

Reply:
0.8.0, it is due to be released later this month: https://projects.eclipse.org/projects/technology.lsp4j/releases/0.8.0

Reply:
```suggestion
 * LSP4J 0.8.x &rarr; DAP 1.32.0
```

Reply:
OK, set to 0.8.x